README.md

```diff
@@ -3,42 +3,72 @@
+## Documentation
+
++ [Features](#features)
++ [Getting Started](#getting-started)
++ [How it Works](#how-it-works)
+    - [Client-Side Rendering vs. Server-Side Rendering](#client-side-rendering-vs-server-side-rendering)
+    - [Building the Bundles](#building-the-bundles)
+    - [Globally Exposing Your React Components](#globally-exposing-your-react-components)
+    - [Rails View Helpers In-Depth](#rails-view-helpers-in-depth)
++ [Generator](#generator)
+    - [Understanding the Organization of the Generated Client Code](#understanding-the-organization-of-the-generated-client-code)
+    - [Redux](#redux)
+    - [Using Images and Fonts](#using-images-and-fonts)
+    - [Bootstrap Integration](#bootstrap-integration)
+        + [Bootstrap via Rails Server](#bootstrap-via-rails-server)
+        + [Bootstrap via Webpack HMR Dev Server](#bootstrap-via-webpack-hmr-dev-server)
+        + [Keeping Custom Bootstrap Configurations Synced](#keeping-custom-bootstrap-configurations-synced)
+    - [Linters](#linters)
+        + [JavaScript Linters](#javascript-linters)
+        + [Ruby Linters](#ruby-linters)
+        + [Running the Linters](#running-the-linters)
++ [Developing with the Webpack Dev Server](#developing-with-the-webpack-dev-server)
++ [Adding Additional Routes for the Dev Server](#adding-additional-routes-for-the-dev-server)
++ [Additional Reading](#additional-reading)
++ [Contributing](#contributing)
++ [License](#license)
++ [Authors](#authors)
++ [About ShakaCode](#about-shakacode)
+
+---
+
+## Features
+Like the [react-rails](https://github.com/reactjs/react-rails) gem, React on Rails is capable of server-side rendering with fragment caching and is compatible with [turbolinks](https://github.com/rails/turbolinks). Unlike react-rails, which depends heavily on sprockets and jquery-ujs, React on Rails uses [webpack](http://webpack.github.io/) and does not depend on jQuery. While the initial setup is slightly more involved, it allows for advanced functionality such as:
 
-In the following screenshot you can see the actual HTML rendered for a side-by-side comparison of a React component left as JavaScript for the client to render followed by the same component rendered on the server to HTML along with any console error messages generated:
++ [Redux](https://github.com/rackt/redux)
++ [Webpack dev server](https://webpack.github.io/docs/webpack-dev-server.html) with [hot module replacement](https://webpack.github.io/docs/hot-module-replacement-with-webpack.html)
++ [Webpack optimization functionality](https://github.com/webpack/docs/wiki/optimization)
++ _**(Coming Soon)**_ [React Router](https://github.com/rackt/react-router)
```

Comment:
[Gem Pull Request](https://github.com/shakacode/react_on_rails/pull/68) and [Tutorial Pull Request](https://github.com/shakacode/react-webpack-rails-tutorial/pull/128).